### PR TITLE
[ticket/16686] Simplify get_database_size() SQL query for PostgreSQL

### PR DESCRIPTION
**Pull request by AlfredoRamos**
Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

[PHPBB3-16686](https://tracker.phpbb.com/browse/PHPBB3-16686)

### CODE REVIEW

File: phpBB/includes/functions_admin.php [hunk 1]
```diff
@@ -2900,37 +2900,18 @@ function get_database_size()
...
-
-				$oid = $row['oid'];
-
-				$sql = 'SELECT pg_database_size(' . $oid . ') as size';
-				$result = $db->sql_query($sql);
-				$row = $db->sql_fetchrow($result);
-				$db->sql_freeresult($result);
-
-				$database_size = $row['size'];
+				list($database, ) = explode('.', $database);
```
Comment (CHItA) [inline phpBB/includes/functions_admin.php:2907]: just `$database = explode('.', $database)[0]`?

Comment (AlfredoRamos) [reply phpBB/includes/functions_admin.php:2907]: Yes, I wasn't sure about changing the old code.

I'll change that.

**Comment by AlfredoRamos** [comment]
Most tests [passed](https://github.com/phpbb/phpbb/pull/6122/checks?sha=1597552b9453338b46d457211ba12193de04f9c9), including the [one for PostgreSQL](https://github.com/phpbb/phpbb/pull/6122/checks?check_run_id=1710598213).

The ones that failed are due to timeouts in the UCP, unrelated to this PR.

- https://github.com/phpbb/phpbb/pull/6122/checks?check_run_id=1710598041#step:12:344
- https://github.com/phpbb/phpbb/pull/6122/checks?check_run_id=1710597757#step:10:113
- https://github.com/phpbb/phpbb/pull/6122/checks?check_run_id=1710597405#step:10:344

**Comment by CHItA** [comment]
Yeah, github seemed to have some hick ups.

**Review by marc1706 — APPROVED** [review]

**Pull request merged**